storages/response_picker.go

```diff
@@ -164,8 +170,37 @@ func (drp *deleteResponsePicker) pullResponses(out chan<- BackendResponse) {
 	close(drp.syncLogReady)
 }
 
+
+func pullResponsesWatchdog(drp *baseDeleteResponsePicker, out chan<- BackendResponse) {
```

Comment:
Why we have different behaviour for watched requests?

Reply:
Because the current picker requires a DELETE request to be successful on all of the backed.